packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -306,9 +297,8 @@ module.exports = (themeOptions = {}) => {
       },
 
       /**
-       * The following plugins need to be last
+       * The following plugin need to be last
        */
-      'gatsby-plugin-remove-trailing-slashes',
       'gatsby-plugin-meta-redirect',
```

Comment:
@timonrey  i know it's merged now but as far as I know the meta redirect plugin can also be removed now.  We're not using the function anywhere - at least I was not able to find it.